ytfzf

```diff
@@ -89,12 +89,16 @@ pid_file="$cache_dir/.pid"
 [ -d "$thumb_dir" ] || mkdir -p "$thumb_dir"
 
 #> config settings
-#list of shortcuts to use in fzf
-#handle_custom_shortcuts will be called if defined (handle_custom_shortcuts can call handle_shortcuts to handle normal shortcuts)
-shortcuts="${shortcuts-alt-l,alt-t,alt-o,alt-v,alt-d,alt-m,alt-s,alt-enter}"
+#handle_custom_key_binds will be called if defined (handle_custom_key_binds can call handle_key_binds to handle normal key binds)
+# key_binds sets the bindings to use for fzf
+# any undefined binds will be used to select
+key_binds="${key_binds-alt-l,alt-t,alt-o,alt-v,alt-d,alt-m,alt-s,alt-enter,enter,double-click}"
+# these keys allow the use of key_binds without leaving fzf
+persistent_key_binds="${persistent_key_binds-enter,alt-enter,alt-v,alt-d,alt-o,double-click}"
```

Comment:
what is the point of `persistent_key_binds` seems a bit redundant?

Reply:
Because you don't want every key bind to be persistent, only some.

Reply:
keys defined in `persistent_key_binds` seem to do nothing, i commented it out, and they keys did the same thing?

Reply:
I don't understand, I just did the same and it works perfectly fine?

Reply:
Try the same experiment, but without looping, or watch if the selection gets reset to the beginning without `persistent_key_binds`

Reply:
for me it just doesn't keep fzf open, which i'm assuming it's supposed to do. Lemme try without my config file.

Reply:
That's odd?
I've tried it with and without my config and it works fine.
What shell are you using?

Reply:
my /bin/sh is dash

Reply:
That's the shell I'm using, just in case I just tried it with bash and it also works there.
It should work.

Are you sure you tried it without your config?

Reply:
No, no i got it to work.